catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/OrderRequestImpl.java

```diff
@@ -216,23 +216,23 @@ private void requestProductResource(List<ResourceContainer> files, ProductDetail
   }
 
   @Override
-  public RequestDescription get_request_description() throws ProcessingFault, SystemFault {
```

Comment:
❗️ You can't rename these methods. They override interface methods in generated CORBA api code.